.github/workflows/tag_release.yml

```diff
@@ -46,15 +46,11 @@ jobs:
     - name: Commit CHANGELOG.md
       env:
         GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
-        COMMIT_MSG: |
-          Release ${GIT_TAG}
-
-
-          skip-checks: true
+        COMMIT_MSG: Release ${GIT_TAG}
       run: |
         git config user.email "oss@fastly.com"
         git config user.name "Release bot"
         git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
         git checkout master
         git add CHANGELOG.md
-        git diff --quiet && git diff --staged --quiet || (git commit -m "${COMMIT_MSG}"; git push origin master)
+        git diff --quiet && git diff --staged --quiet || (git commit -m "${COMMIT_MSG}"; git push --force origin master)
```

Comment:
this has the potential of accidentally wiping out merges to master that happen in quick succession. I'm not sure that the `--force` flag actually bypasses branch protections either.